### DISCUSSION TITLE
Please, make age usable or remove the "UNIX-style" from the description

### DISCUSSION THREAD

**Discussion by clydeconfigs** [FiloSottile/age#546] [UX reports]
my problem comes when using the commands and having to follow a strict order, for whatever reason. i think it has to do with the go parser, but who cares. fix this and let me use whatever order of arguments i want

i cannot do:
`age -e secret -o secret.age`

which is extremely obvious that that should work, right? and i should also be able to just use 

`age -e secret`

and get a simple `secret.age` output, so i don't have to burn my fingers typing...;

`age -r age1kgaxf9udncjdu2qfd9xe4ylz28r7gvded9kk7xcnnwhmwc9rda8qn9pc2m -o secret.age secret`

that is not comfortable.

the command parser, at this time, is UNUSABLE, NOT unix based, and should be fixed. get to work those who know how to change this, or i will do it myself

i love the concept, but this makes me angry >:(

**Comment by FiloSottile** [comment]
I don't appreciate the tone. You're welcome to use a different implementation, maintain your own fork, or not use age.
